pulsar-client-admin-api/src/main/java/org/apache/pulsar/common/policies/data/impl/BacklogQuotaImpl.java

```diff
@@ -80,12 +80,12 @@ public void setLimitSize(long limitSize) {
         this.limit = limitSize;
     }
 
-    public int getLimitTime() {
-        return limitTime;
+    public int getLimitTimeInSec() {
```

Comment:
This is a breaking change without any backwards compatibility.

Reply:
I think it's better to revert this change and instead focus on improving documentation.

Reply:
I agree with that approach. There are many names in Pulsar that are slightly confusing. It'd make more sense to define a paradigm for naming before making many one-off changes.